Project.toml

```
@@ -12,12 +12,12 @@ Roots = "f2b01f46-fcfa-551c-844a-d8ac1e96c665"
 StructArrays = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
 
 [compat]
-Adapt = "3.4"
-JLD2 = "0.4.31"
+Adapt = "3"
+JLD2 = "^0.4"
```

```suggestion
JLD2 = "0.4"
```

(it's the same, right?)

---

Oh yeah think so!